index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio - ZC</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./icons/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <!-- Toolbar -->
+    <div class="toolbar">
+        <p id="toolbartext">Welcome</p>
+        <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+    </div>
+
+    <!-- Headline Section-->
+    <div class="headline">
+
+        <div class="container">
+            <h1>Hey there. I'm Cristian<br><span style="color: #36B37E"> I'm a software developer</span> </h1>
+        </div>
+
+        <div class="container">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+        <div class="container">
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/twitter.svg" alt="Twitter">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/linkedinicon.svg" alt="LinkedIn">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/medium.svg" alt="Medium">
+            </div>
+
```

Comment:
- [ ] Kindly 😃The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.💯


| social |
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/219174435-8637bdf3-5a0c-4c4c-b994-5c290d8686b5.png) |